packages/kit/src/views/Send/pages/SendConfirm/TxAdvancedSettingsContainer.tsx

```
@@ -0,0 +1,336 @@
+import { useCallback, useEffect, useMemo, useState } from 'react';
+
+import BigNumber from 'bignumber.js';
+import { utils } from 'ethers';
+import { isNaN, isNil } from 'lodash';
```

Biome

[error] Do not shadow the global "isNaN" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)